products/dns/api.yaml

```diff
@@ -81,6 +83,17 @@ objects:
           The time that this resource was created on the server.
           This is in RFC3339 text format.
         output: true
+      - !ruby/object:Api::Type::KeyValuePairs
+        name: 'labels'
+        description: |
+          A set of key/value label pairs to assign to this ManagedZone.
+        update_verb: :PATCH
```

Comment:
This and the next line can be removed, since it's the same as the rest of the resource

Reply:
From your comments I get that I should keep the verb but remove the URL, please let me know otherwise.

Reply:
nope, remove both the verb and the URL.